content/release-10-14-0/announcements-10-14-0.md

```diff
@@ -105,6 +105,8 @@ If your microservice is using the deprecated API version 1 and is deployed to a
 ### SDK changes
 
 #### Planned
+#### Planned
+As of release 10.16 we plan to update the Web SDK to Angular 14. See the [Angular upgrade guide](https://v14.angular.io/guide/update-to-latest-version) for more information on what needs to be changed in your custom application. You can use our [default upgrade guide](https://cumulocity.com/guides/web/upgrade/#update-to-an-newer-version) to update the files provided by the Cumulocity cli tool.
```

Comment:
@BeateRixen just a general question: Do we format "cli" in capital letters or in lower case (as shown here)?

Reply:
We spell it CLI. I edited the text already.